src/communication/stmt.rs

```diff
@@ -0,0 +1,153 @@
+use super::CommunicationReplacer;
+use crate::comm_repl_default_impl;
+use crate::{ast::*, ClassOrInterfaceComponent};
+use crate::{prophet::ModuleComponent, MethodComponent};
+
+comm_repl_default_impl!(
+    ReturnStmt,
```

Comment:
Couldn't a Return statement return the result of an RPC/Rest call directly and thus be missed by using the defeault implementation here?

Reply:
Good call, i'll fix this.